intercom_flutter/ios/Classes/IntercomFlutterPlugin.m

```diff
@@ -31,6 +31,9 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
         [[UnreadStreamHandler alloc] init];
     [unreadChannel setStreamHandler:unreadStreamHandler];
     
+    #if DEBUG
+    [Intercom enableLogging];
+    #endif
```

Comment:
This should not be here. Basically, there has to be a method to enable the logging.

Reply:
Oops, sure, will remove this commit